core/src/main/java/fr/sncf/osrd/App.java

```diff
@@ -16,6 +16,7 @@ public static void main(String[] args) {
         commands.put("pretty-print-signals", new PrettyPrintCommand());
         commands.put("api", new ApiServerCommand());
         commands.put("interactive", new InteractiveCommand());
+        commands.put("test-infra", new TestInfraCommand());
```

Comment:
I have doubts about having this as a command. We could have the `TestInfraCommand` functions in the test framework that runs with tests.

Reply:
It isn't exactly a test that would make sense in the framework, it can't really pass or fail, it's more a visualization tool